src/styles/_markdown.scss

```diff
@@ -0,0 +1,41 @@
+// Styles for overview and guide docs generated via `marked` from the material2 repo.
+
+.docs-markdown-h1 {
+  display: inline-block;
+  font-size: 20px;
+  font-weight: 400;
+  padding: 5px;
+}
+
+.docs-markdown-h2 {
```

Comment:
Unless your planning on having other styles here, you can merge the `.docs-markdown-h2` and `.docs-markdown-h4` selectors.